src/EPR.Payment.Service.Common.UnitTests/Mocks/MockIRegistrationFeesRepository.cs

```diff
@@ -125,6 +125,16 @@ public static class MockIRegistrationFeesRepository
                 EffectiveFrom = DateTime.UtcNow.AddDays(-10), // Effective 10 days ago
                 EffectiveTo = DateTime.UtcNow.AddDays(10) // Expires in 10 days
             },
+            // Late Fee
+            new Common.Data.DataModels.Lookups.RegistrationFees
+            {
+                Group = new Common.Data.DataModels.Lookups.Group { Type = GroupTypeConstants.ProducerType, Description = "Producer Type" },
+                SubGroup = new Common.Data.DataModels.Lookups.SubGroup { Type=SubGroupTypeConstants.LateFee, Description = "Late Fee" },
+                Regulator = new Common.Data.DataModels.Lookups.Regulator { Type="GB-ENG", Description = "England" },
```

Comment:
Same as above here. Very minor but I think it is good to keep format consistent.